crates/ruma-events/src/pdu.rs

```diff
@@ -152,7 +153,15 @@ pub struct RoomV3Pdu {
 
 /// Content hashes of a PDU.
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
 pub struct EventHash {
     /// The SHA-256 hash.
     pub sha256: String,
 }
+
+impl EventHash {
+    /// Create a new `EventHash` with the given SHA256 hash.
+    pub fn new(sha: String) -> Self {
```

Comment:
Why name this `sha` instead of `sha256`?

Reply:
Ehh I just wasn't crazy about the numbers in the parameter name 🤷 but I changed it